lib/y2_module_basetest.pm

```diff
@@ -106,6 +107,21 @@ sub workaround_suppress_lvm_warnings {
     }
 }
 
+=head2 use_wicked_network_manager
+
+ use_wicked_network_manager();
+
+switch network manager to wicked.
+
+=cut
+sub use_wicked_network_manager {
```

Comment:
We have `tests/console/switch_to_wicked.pm` module already. Do you think we could have extracted common code?
Also, I think `consoletest` is better place for it as wicked is not related to YaST modules directly. What do you say?

Reply:
Good catch. and it makes totally sense